cirq/study/visualize.py

```diff
@@ -16,10 +16,10 @@
 
 import numpy as np
 
-from cirq.study import trial_result
+from cirq.study import result
```

Comment:
```
if TYPE_CHECKING:
    from cirq.study import result
```

Reply:
Also, you'll need to use single quotes for the type`'result.Result'`  in the `plot_state_histogram` signature below

Reply:
Done.